acados/ocp_nlp/ocp_nlp_constraints_bgp.c

```diff
@@ -1049,7 +1049,6 @@ void ocp_nlp_constraints_bgp_update_qp_matrices(void *config_, void *dims_, void
 
     int nv = nx + nu;
 
-    // XXX large enough ?
```

Comment:
All these comments on `XXX large enough ?` are intended to remind that these quantities have to be checked and updated in case the definition of the external function changes, since this is something very easy to overlook (and it indeed happened in the past).

You can replace them with a nicer statement if you like, but I would leave some note about this.

Reply:
I think it is quite obvious that these would have to be changed when the external function expected by the module changes.
I checked that all these sizes with respect to how it is used..